client/implementation/src/main/java/io/smallrye/graphql/client/typesafe/impl/RequestBuilder.java

```diff
@@ -16,10 +19,13 @@ public RequestBuilder(MethodInfo method) {
 
     String build() {
         request.append(method.getName());
-        if (method.getParameterCount() > 0) {
+        List<ParameterInfo> parameters = method.parameters()
+                .filter(parameterInfo -> !parameterInfo.isAnnotated(Header.class))
```

Comment:
This won't collect parameters annotated with `AuthorizationHeader`.  Is that intended?

Reply:
Yes. Username/Password would require two fields, so the `AuthorizationHeader` annotation doesn't work on a single parameter. It also doesn't work with methods, just with config. They are quite different concepts.